mathesar/tests/integration/conftest.py

```diff
@@ -130,3 +131,19 @@ def go_to_patents_data_table(page, create_table, schema_name, base_schema_url):
     page.goto(base_schema_url)
     get_table_entry(page, table_name).click()
     yield table_name
+
+
+@pytest.fixture
+def go_to_table_with_numbers_in_text(page, patent_schema, create_column, schema, base_schema_url):
```

Comment:
``` python
def go_to_table_with_numbers_in_text(page, patent_schema, create_column, schema, base_schema_url):
```
In this line it will be more appropriate to use `custom_types_schema` instead of `patent_schema`, once the PR #1199 is merged.